os/include/sys/sock_internal.h

```diff
@@ -217,6 +218,16 @@ struct linger {
 	int l_linger;				/* Linger time, in seconds. */
 };
 
+struct msghdr {
```

Comment:
> Socket_utils_common_posix.cc (external\grpc\src\core\lib\iomgr):#include <sys/sock_internal.h>
> Tcp_posix.cc (external\grpc\src\core\lib\iomgr):#include <sys/sock_internal.h>

Because they include both `sock_internal.h` and `socket.h`, I think this causes redefine errors.

Reply:
That is right. So I have changed definition of struct msghdr into sys/socket.h. Now it seems work. Please let me know your opinion. Thank you.